eo-maven-plugin/src/test/java/org/eolang/maven/ProbeMojoTest.java

```diff
@@ -122,6 +162,134 @@ void findsProbesInOyRemote(@TempDir final Path temp) throws IOException {
+                "+package org.eolang.custom\n",
+                "[] > main",
+                "  QQ.io.stdout|0.28.10 > @",
+                "    \"Hello world\""
+            )
+            .execute(new FakeMaven.Probe());
+        MatcherAssert.assertThat(
+            String.format(
+                "Tojos should have contained versioned object %s after probing, but they didn't",
+                object
+            ),
+            maven.externalTojos().contains(object),
+            Matchers.is(true)
+        );
+        MatcherAssert.assertThat(
+            "First entry of tojos after probing should have contained one probed object, but it didn't",
+            ProbeMojoTest.firstEntry(
+                maven.externalPath(),
+                "probed"
+            ),
+            Matchers.equalTo("1")
+        );
+    }
+
+    @Test
+    @ExtendWith(OnlineCondition.class)
+    @Disabled
+    void findsProbesWithVersionsInDifferentObjectionaries(@TempDir final Path temp)
+        throws IOException {
+        final CommitHash first = new CommitHashesMap.Fake().get("0.28.10");
+        final CommitHash second = new CommitHashesMap.Fake().get("0.28.0");
+        final String stdout = "org.eolang.io.stdout|9b88393";
+        final String sprintf = "org.eolang.txt.sprintf|6a70071";
+        final FakeMaven maven = new FakeMaven(temp)
+            .with(
+                "objectionaries",
+                new OjsDefault()
+                    .with(first, new OyRemote(first))
+                    .with(second, new OyRemote(second))
+            )
+            .with("withVersions", true)
+            .withProgram(
+                "+package org.eolang.custom\n",
+                "[] > main",
+                "  QQ.io.stdout|0.28.10 > @",
+                "    QQ.txt.sprintf|0.28.0",
+                "      \"Hello world!\""
+            )
+            .execute(new FakeMaven.Probe());
+        MatcherAssert.assertThat(
+            String.format(
+                "Tojos should have contained versioned object %s after probing, but they didn't",
+                stdout
+            ),
+            maven.externalTojos().contains(stdout),
+            Matchers.is(true)
+        );
+        MatcherAssert.assertThat(
+            String.format(
+                "Tojos should have contained versioned object %s after probing, but they didn't",
+                sprintf
+            ),
+            maven.externalTojos().contains(stdout),
+            Matchers.is(true)
+        );
+        MatcherAssert.assertThat(
+            "First entry of tojos after probing should have contained two probed objects, but it didn't",
+            ProbeMojoTest.firstEntry(
+                maven.externalPath(),
+                "probed"
+            ),
+            Matchers.equalTo("2")
+        );
+    }
+
+    @Test
+    @ExtendWith(OnlineCondition.class)
+    @Disabled
+    void findsProbesWithDefaultHash(@TempDir final Path temp) throws IOException {
```

Comment:
@volodya-lombrozo I don't know (at least for now) how to resolve that, because tests are similar but not identical. Every test has something a bit different from other one. I suggest to add a puzzle for that